{"name": "test_banners[chromium]", "status": "failed", "statusDetails": {"message": "AssertionError", "trace": "self = <Suites.Banners.banners.BannersZeroDep object at 0x0000021FFC882AB0>\n\n    @allure.step('Confirm login')\n    def press_confirm_log_in_button(self):\n        try:\n>           self.confirm_log_in_button.click()\n\nSuites\\Base\\BaseSetUp.py:55: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n.venv\\Lib\\site-packages\\playwright\\sync_api\\_generated.py:14851: in click\n    self._sync(\n.venv\\Lib\\site-packages\\playwright\\_impl\\_locator.py:153: in click\n    return await self._frame.click(self._selector, strict=True, **params)\n.venv\\Lib\\site-packages\\playwright\\_impl\\_frame.py:488: in click\n    await self._channel.send(\"click\", locals_to_params(locals()))\n.venv\\Lib\\site-packages\\playwright\\_impl\\_connection.py:59: in send\n    return await self._connection.wrap_api_call(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <playwright._impl._connection.Connection object at 0x0000021FFBD93B00>, cb = <function Channel.send.<locals>.<lambda> at 0x0000021FFC70AB60>, is_internal = False\n\n    async def wrap_api_call(\n        self, cb: Callable[[], Any], is_internal: bool = False\n    ) -> Any:\n        if self._api_zone.get():\n            return await cb()\n        task = asyncio.current_task(self._loop)\n        st: List[inspect.FrameInfo] = getattr(task, \"__pw_stack__\", inspect.stack())\n        parsed_st = _extract_stack_trace_information_from_stack(st, is_internal)\n        self._api_zone.set(parsed_st)\n        try:\n            return await cb()\n        except Exception as error:\n>           raise rewrite_error(error, f\"{parsed_st['apiName']}: {error}\") from None\nE           playwright._impl._errors.TimeoutError: Locator.click: Timeout 30000ms exceeded.\nE           Call log:\nE           waiting for locator(\"//button[contains(@class, 'min-w-full') and contains(text(),'Log in')]\")\n\n.venv\\Lib\\site-packages\\playwright\\_impl\\_connection.py:513: TimeoutError\n\nThe above exception was the direct cause of the following exception:\n\npage = <Page url='https://tombriches.com/'>\n\n    @allure.suite(\"Banners test\")\n    def test_banners(page):\n        banners = BannersZeroDep(page)\n>       banners.set_up()\n\ntest_tr_smoke.py:61: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\nSuites\\Base\\BaseSetUp.py:81: in set_up\n    self.press_confirm_log_in_button()\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <Suites.Banners.banners.BannersZeroDep object at 0x0000021FFC882AB0>\n\n    @allure.step('Confirm login')\n    def press_confirm_log_in_button(self):\n        try:\n            self.confirm_log_in_button.click()\n            time.sleep(5)\n            allure.attach(self.page.screenshot(), name='Log in button pressed',\n                          attachment_type=allure.attachment_type.PNG)\n            self.page.goto('https://tombriches.com/')\n    \n        except Exception as e:\n            allure.attach(self.page.screenshot(), name='Log in button is not pressed',\n                          attachment_type=allure.attachment_type.PNG)\n>           raise AssertionError from e\nE           AssertionError\n\nSuites\\Base\\BaseSetUp.py:64: AssertionError"}, "steps": [{"name": "Press on Log in button", "status": "passed", "attachments": [{"name": "Log in button pressed", "source": "e7f1348c-f479-4b50-aabf-8742cf850033-attachment.png", "type": "image/png"}], "start": 1714566466420, "stop": 1714566466757}, {"name": "Enter credentials", "status": "passed", "attachments": [{"name": "E-mail field filled", "source": "d2496c1c-b6f5-4214-83f3-7f0beb74e8cd-attachment.png", "type": "image/png"}, {"name": "Password field filled", "source": "d227d4e4-c7d7-40ef-ba73-49a7f4242a89-attachment.png", "type": "image/png"}], "start": 1714566466757, "stop": 1714566467126}, {"name": "Confirm login", "status": "failed", "statusDetails": {"message": "AssertionError\n", "trace": "  File \"C:\\Users\\User\\Desktop\\Automation\\TR_Smoke\\.venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"C:\\Users\\User\\Desktop\\Automation\\TR_Smoke\\Suites\\Base\\BaseSetUp.py\", line 64, in press_confirm_log_in_button\n    raise AssertionError from e\n"}, "attachments": [{"name": "Log in button is not pressed", "source": "78b61c03-ee3b-4906-86a8-3a84d4244fd0-attachment.png", "type": "image/png"}], "start": 1714566467126, "stop": 1714566497299}], "attachments": [{"name": "Page open", "source": "360efab0-67ac-4865-84fb-5681647dbae9-attachment.png", "type": "image/png"}], "parameters": [{"name": "browser_name", "value": "'chromium'"}], "start": 1714566464017, "stop": 1714566497299, "uuid": "d4903229-20dc-4638-876f-b9ef54b6dcba", "historyId": "241f545a2af84c56a9c1b7dea72cf699", "testCaseId": "70cc32a6367a0b96ec43ad6fee71106d", "fullName": "test_tr_smoke#test_banners", "labels": [{"name": "suite", "value": "Banners test"}, {"name": "host", "value": "DESKTOP-EFHMBBJ"}, {"name": "thread", "value": "25980-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_tr_smoke"}]}